packages/react/src/components/cookieConsentCore/cookieHandler.js

```diff
@@ -466,16 +466,10 @@ export default class CookieHandler {
     }
 
     const siteSettingsGroups = [...this.#siteSettings.requiredGroups, ...this.#siteSettings.optionalGroups];
+    const groupIds = siteSettingsGroups.map((group) => group.groupId);
+    const duplicateGroupNames = groupIds.filter((groupId, index) => groupIds.indexOf(groupId) !== index);
 
```

Comment:
I think the fastest would be

```
const uniqueGroupIds = new Set(groupIds); // or(...groupIds)
if(uniqueGroupIds.size != groupIds.length){....}
```

Reply:
Hah thought of the exact same thing later but didn't change it :D